tools/wpt/browser.py

```diff
@@ -745,8 +745,11 @@ def install_webdriver_by_version(self, version, dest=None):
             os.chmod(existing_binary_path, stat.S_IWUSR)
             os.remove(existing_binary_path)
 
-        url = self._latest_chromedriver_url(version) if version \
-            else self._chromium_chromedriver_url(None)
+        # Pinned ChromeDriver downloading URL. TODO(kyle).
+        # See https://github.com/web-platform-tests/wpt/issues/30090.
+        # url = self._latest_chromedriver_url(version) if version \
+        #     else self._chromium_chromedriver_url(None)
+        url = "https://storage.googleapis.com/chromium-browser-snapshots/Linux_x64/913061/chromedriver_linux64.zip"
```

Comment:
This is using the Linux binary for all platforms, and I think that's why macOS and Windows tests are failing now.

Can we instead hardcode `version` to something that will result in a working URL to download on all platforms?

Reply:
Per our offline discussion, we will try pinning a version of Chrome that will download a working version of ChromeDriver